launch/gazebo.launch

```diff
@@ -1,20 +1,24 @@
 <?xml version="1.0" ?>
 <launch>
+    <arg name="__ns" default=""/>
     <arg name="world" default="worlds/empty.world" />
     <arg name="x_pos" default="0"/>
     <arg name="model" default="robotont_gazebo_nuc"/>
+    <arg name="robot_name" default="robotont"/>
+    <arg name="cam_fps" default="5"/>

```

Comment:
these two arguments appear to be unused. am i missing something?

Reply:
robot name has moved to spawn_urdf, not really sure if the param should be passed through e.g. world_minimaze->gazebo->spawn_urdf launch file includes
cam fps was before used for throttling but was probably discarded since the simulation is typically running locally.